javalib/src/main/scala/java/io/File.scala

```diff
@@ -836,8 +836,10 @@ object File {
 
         // found an absolute path. continue from there.
         case link if link(0) == separatorChar =>
-          if (Platform.isWindows() && strncmp(link, c"\\\\?\\", 4.toUInt) == 0)
-            path
+          if (isWindows)
```

Comment:
It may be possible to remove the include of `Platform`.
I was moving quickly, but I did not see any other uses of it in the this new file.